cql/ManageCommonAbnormality.cql

```diff
@@ -462,7 +462,8 @@ define ErrataRecommendation:
         short: 'Colposcopy',
         date: Today(),
         details: {
-          'Colposcopy is recommended.'
+          'Colposcopy is recommended.',
+          'If a patient has a negative HPV test or co-test following a low-grade result for which colposcopy was previously recommended but not performed, repeating an HPV test or cotest in 1 year is acceptable. (Perkins 2019 Erratum)'
         }
       }
```

Comment:
Why exactly is this being added here? If a patient's most recent HPV test or cotest were negative, they wouldn't fall into this recommendation, right?

Reply:
We chatted on Slack, and decided that this new text would apply only to 3.2.2.a, and not to 3.2.2.b, since 3.2.2.b deals with patients who have successive HPV+ results, which should result in a colposcopy recommendation.